FFXIVClientStructs/FFXIV/Client/UI/RaptureAtkModule.cs

```diff
@@ -43,10 +43,10 @@ public unsafe partial struct RaptureAtkModule {
     public partial bool DecRefStringArrayData(int index);
 
     [MemberFunction("E8 ?? ?? ?? ?? 66 89 46 50")]
-    public partial nint OpenAddon(uint addonNameId, uint valueCount, AtkValue* values, AgentInterface* parentAgent, long unk, uint ownerAddonId, int unk2);
+    public partial ushort OpenAddon(ushort addonNameId, uint valueCount, AtkValue* values, AgentInterface* parentAgent, ulong unk, ushort parentAddonId, int unk2);
```

Comment:
No, `addonNameId` should stay `uint`. There are currently 852 addon names it can pick from.

Reply:
Oh right, lol. It's used as uint, but whatever I guess?

Reply:
I am not sure what happens on a CPU. Like, does it only write 2 bytes into the register if we specify it as ushort? That could be problematic if it reads an uint afterwards?!